.gitattributes

```diff
@@ -1,33 +1,2 @@
 # Auto detect text files and perform LF normalization
-* text=auto
```

Comment:
Could you explain why we are removing these lines? ...A better question, do we want to remove all of them?

Reply:
we shouldn't be normalizing anything files should be left alone as is so they can be edited in all platforms. There is no need to specify individual extensions, wild card specifies all should be left alone.